ucsb-cs156-f23/team03-f23-5pm-4
# Recommendation Request Create Page

Closes #31 

<img width="1397" alt="Screenshot 2023-11-09 at 2 43 55 PM" src="https://github.com/ucsb-cs156-f23/team03-f23-5pm-4/assets/91707453/c879462a-d091-488b-9195-22947ed7f976">

Testing instructions:

log into [dokku deployment](  https://team03-aditiphatak-dev.dokku-04.cs.ucsb.edu) 
navigate [recommendation requests](https://team03-aditiphatak-dev.dokku-04.cs.ucsb.edu/recommendationrequest) 
click create and  input fields and press create
navigate to [table](http://localhost:8080/swagger-ui/index.html) 
verify that GET all endpoint includes the item you just created
